lib/apiservers/service/restapi/handlers/decode/networking.go

```diff
@@ -76,9 +77,21 @@ func FromGateway(m *models.Gateway) string {
 func FromImageFetchProxy(p *models.VCHRegistryImageFetchProxy) common.Proxies {
 	http := string(p.HTTP)
 	https := string(p.HTTPS)
+	var nproxy *string
+	if p.NoProxy != nil {
+		var buffer bytes.Buffer
+		buffer.WriteString(string(p.NoProxy[0]))
+		for _, v := range p.NoProxy[1:] {
+			buffer.WriteString(",")
+			buffer.WriteString(string(v))
+		}
+		nproxyStr := buffer.String()
```

Comment:
I think this could be more clearly written as `strings.Join(p.NoProxy, ",")`

Reply:
I tried initially, but looks p.NoProxy is []strfmt.URL type, and strings.Join does not accept it?